sdk/openai/openai/test/public/utils/models.ts

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+export const visionModels = [{ name: "gpt-4", version: "vision-preview" }];
+export const whisperModels = [{ name: "whisper", version: "001" }];
+export const functionCallModels = [
+  { name: "gpt-4", version: "0125-preview" },
+  { name: "gpt-4", version: "0613" },
+  { name: "gpt-35-turbo", version: "0613" },
+  { name: "gpt-35-turbo-16k", version: "0613" },
+];
+export const imageGenerationModels = [{ name: "dall-e-3", version: "3.0" }];
+export const embeddingModels = [
+  { name: "text-embedding-3-small", version: "1" },
+  { name: "text-embedding-ada-002", version: "2" },
+];
```

Comment:
can we filter out models that don't work so we don't have to worry about adding new models? We can limit the list to models that don't return expected errors in unsupported usage scenarios.